pywbemtools/pywbemcli/pywbemcli.py

```diff
@@ -322,9 +322,15 @@ def set_default_if_empty_str(tst_str, default=None):
                    u'Default: EnvVar {ev}, or {default}'.
                    format(ev=PywbemServer.pull_max_cnt_envvar,
                           default=DEFAULT_MAXPULLCNT))
-@click.option('-T', '--timestats', is_flag=True,
+@click.option('-T', '--timestats',
+              type=click.Choice(["auto", "cmd", "none"]),
+              default=None,
               # defaulted in code
-              help=u'Show time statistics of WBEM server operations.')
+              help=u'Show time statistics of WBEM server operations and define '
+              u'when displayed. '
+              u'"auto" - display after each command. '
+              u'"cmd"  - display only with "statistics show" command. '
+              u'"none" - allows the option to be disabled in interactive mode.')
```

Comment:
The help text does not state what the default value is.

Reply:
Added default and also added envvar since that had been left off before.